jjb/dynamic/scale-ci_deployments_per_ns.yml

```diff
@@ -0,0 +1,181 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+
+        # Disable logging
+        set +x
+        oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        # Re-enable logging
+        set -x
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        export PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        export ANSIBLE_FORCE_COLOR=true
+        ansible --version
+        time ansible-playbook -vv -i inventory workloads/cluster-limits-deployments-per-ns.yml
+
+        oc logs --timestamps -n scale-ci-tooling -f job/scale-ci-deployments-per-ns
+
+        oc get job -n scale-ci-tooling scale-ci-deployments-per-ns -o json | jq -e '.status.succeeded==1'
+    concurrent: true
+    description: |
+      Deployments per namespace workload is used to validate and push the cluster limits of OpenShift. This loads up the cluster with maximum deployments per namespace using cluster loader.
+      This job is managed by https://github.com/openshift-scale/scale-ci-pipeline
+    disabled: false
+    name: ATS-SCALE-CI-DEPLOYMENTS-PER-NAMESPACE
+    node: scale-ci
+    parameters:
+    - string:
+        default: "kubeadmin"
+        description: User name to access cluster.
+        name: CLUSTER_USER
+    - string:
+        default: ""
+        description: Password for CLUSTER_USER to access cluster.
+        name: CLUSTER_PASSWORD
+    - string:
+        default: ""
+        description: The URL address to the openshift cluster to login to.
+        name: CLUSTER_API_URL
+    - string:
+        default: ""
+        description: Token to access private repo containing ssh keys.
+        name: SSHKEY_TOKEN
+    - string:
+        default: "localhost"
+        description: The machine intended to run the oc commands and launch the workload.
+        name: ORCHESTRATION_HOST
+    - string:
+        default: "root"
+        description: The user for the Orchestration host.
+        name: ORCHESTRATION_USER
+    - string:
+        default: "quay.io/openshift-scale/scale-ci-workload"
+        description: Container image that runs the workload script.
+        name: WORKLOAD_IMAGE
+    - bool:
+        default: false
+        description: Enables/disables the node selector that places the workload job on the `pbench` node.
+        name: WORKLOAD_JOB_NODE_SELECTOR
```

Comment:
These values are set to `true` for a default right now - https://github.com/openshift-scale/workloads/blob/master/workloads/vars/deployments-per-ns.yml#L17

As I have been adding pass/fail functionality I have been setting as many defaults to support run anywhere vs run with as much instrumentation as possible.  Thus that is why nodevertical/podvertical are set to false for those values

Reply:
Got it, we can stick with false in the templates I guess. I will create a PR to update the workload defaults.